src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2841,6 +2841,209 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
```

Comment:
This should be in alphabetical order between 'if' and 'length'.

Reply:
Fixed.